spec/services/recurring_contribution/subscriptions/create_juntos_spec.rb

```diff
@@ -53,7 +53,7 @@
 
       it "should return a subscription" do
         expect(subject)
-          .to have_attributes(status: :waiting_for_charging_day, plan_id: plan.id, credit_card_key: nil)
+          .to have_attributes(status: 'waiting_for_charging_day', plan_id: plan.id, credit_card_key: nil)
```

Comment:
nao entendi bem o intuito aqui, seria checar se o retorno é `Subscription` ou se o objeto contém o status e id desejado? 
se for só pra checar o tipo, muda o expect pra algo `.is_a(Subscription)` se for o outro caso, muda a definição do `it` 💅